scripts/plugins/utilities.py

```diff
@@ -230,10 +231,12 @@ def Get_Config_List(config_file, schema_file):
         with open(config_file, "r") as stream:
             config_yaml = yaml.load(stream, Loader=yaml.FullLoader)
     except FileNotFoundError as e:
-        logger.error("REQUIRED FILE NOT FOUND: [{}]".format(e.filename))
+        msg, return_number = Get_Doc("missing_file")
+        logger.error("{doc_string} [{missing_file}]".format(doc_string=msg, missing_file=e.filename))
+        logger.debug(e)
+        exit(return_number)
```

Comment:
FYI
```suggestion
        sys.exit(return_number)
```
In the future changes pylint will warn on this. See other changes: https://github.com/bitovi/bitops/pull/298/commits/7b7f9b9aa2f94b9665f924896388ec04abb89fca

NOTE: The highlighted change is not required for this PR, but if you're motivated to change it, - that would be nice so I won't need to update it in the pylint PR #298 again.